$OpenBSD: patch-configure,v 1.15 2020/05/16 08:08:22 steven Exp $

Use GNU readline as our base readline lacks rl_prefer_env_winsize.

Index: configure
--- configure.orig
+++ configure
@@ -67123,13 +67123,13 @@ fi
 
     ac_octave_save_LIBS="$LIBS"
     LIBS="$TERM_LIBS"
-    { $as_echo "$as_me:${as_lineno-$LINENO}: checking for rl_set_keyboard_input_timeout in -lreadline" >&5
-$as_echo_n "checking for rl_set_keyboard_input_timeout in -lreadline... " >&6; }
+    { $as_echo "$as_me:${as_lineno-$LINENO}: checking for rl_set_keyboard_input_timeout in -lereadline" >&5
+$as_echo_n "checking for rl_set_keyboard_input_timeout in -lereadline... " >&6; }
 if ${ac_cv_lib_readline_rl_set_keyboard_input_timeout+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lreadline  $LIBS"
+LIBS="-lereadline  $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -67168,7 +67168,7 @@ fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_lib_readline_rl_set_keyboard_input_timeout" >&5
 $as_echo "$ac_cv_lib_readline_rl_set_keyboard_input_timeout" >&6; }
 if test "x$ac_cv_lib_readline_rl_set_keyboard_input_timeout" = xyes; then :
-  READLINE_LIBS="-lreadline"
+  READLINE_LIBS="-lereadline"
 
 $as_echo "#define USE_READLINE 1" >>confdefs.h
 
